schemas/database/001_initial.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE upstreams (
+	name TEXT PRIMARY KEY,
+	url TEXT UNIQUE NOT NULL,
+	chainid INTEGER NOT NULL
+);
+
+CREATE TABLE signing_keys (
+	id INTEGER PRIMARY KEY
+);
```

Comment:
The keys themselves are stored in AWS HSM, right?

Maybe we want to store their address and the chain (chains?) they are used on too. 🤔

Reply:
definitely makes sense to store both!